<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-TO96F33" failures="0" tests="7" name="tests.CheckoutTest" time="2.174" errors="1" timestamp="2025-10-07T23:02:22 IST" skipped="6">
  <testcase classname="tests.CheckoutTest" name="setup" time="2.174">
    <error message="Item not found: Flat White" type="java.lang.IllegalArgumentException">
      <![CDATA[java.lang.IllegalArgumentException: Item not found: Flat White
at pages.MenuPage.addItemToCart(MenuPage.java:39)
at tests.CheckoutTest.setup(CheckoutTest.java:40)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:71)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurationMethod(ConfigInvoker.java:400)
at org.testng.internal.invokers.ConfigInvoker.invokeConfigurations(ConfigInvoker.java:333)
at org.testng.internal.invokers.TestInvoker.runConfigMethods(TestInvoker.java:836)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:601)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:995)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:741)
at org.testng.TestRunner.run(TestRunner.java:616)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:463)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
]]>
    </error>
  </testcase> <!-- setup -->
  <testcase classname="tests.CheckoutTest" name="verifyPaymentFailsWithEmptyName" time="0.000">
    <skipped/>
  </testcase> <!-- verifyPaymentFailsWithEmptyName -->
  <testcase classname="tests.CheckoutTest" name="verifyPaymentFailsWithInvalidEmail" time="0.000">
    <skipped/>
  </testcase> <!-- verifyPaymentFailsWithInvalidEmail -->
  <testcase classname="tests.CheckoutTest" name="verifyFormIsDisplayed" time="0.000">
    <skipped/>
  </testcase> <!-- verifyFormIsDisplayed -->
  <testcase classname="tests.CheckoutTest" name="verifySuccessfulPayment" time="0.000">
    <skipped/>
  </testcase> <!-- verifySuccessfulPayment -->
  <testcase classname="tests.CheckoutTest" name="verifyPaymentFailsWithBothEmpty" time="0.000">
    <skipped/>
  </testcase> <!-- verifyPaymentFailsWithBothEmpty -->
  <testcase classname="tests.CheckoutTest" name="verifyPaymentFailsWithEmptyEmail" time="0.000">
    <skipped/>
  </testcase> <!-- verifyPaymentFailsWithEmptyEmail -->
</testsuite> <!-- tests.CheckoutTest -->
